bundles/mapping/mapmodule/domain/AbstractLayer.js

```diff
@@ -1211,17 +1213,25 @@ Oskari.clazz.define(
             return this._srs_name;
         },
         /**
-         * @method isSupported does the layer support given projection?
+         * @method isSupportedSrs does the layer support given projection?
          * @param {String} projection
          * @return {Boolean} true if no data about support or param found in supported
          */
-        isSupported: function (projection) {
+        isSupportedSrs: function (projection) {
             if (!this._srsList || !this._srsList.length) {
                 // if list is not provided, treat as supported
                 return true;
             }
             return this._srsList.indexOf(projection) !== -1;
         },
+        /**
+         * @method @deprecated isSupported
+         * @param {String} projection
+         * @return {Boolean} true if no data about support or param found in supported
+         */
+        isSupported: function (projection) {
+            return this.isSupportedSrs(projection);
```

Comment:
Add Oskari.log('AbstractLayer').deprecated() call here so it's easier to spot usage.